Cura/resources/localization-tools/1-generate-pot-file.bat

```diff
@@ -1,5 +1,5 @@
 setlocal EnableDelayedExpansion
 for /f "delims=" %%x in (python_files_to_localize) do set var=!var! ..\..\%%x
 echo !var!
-pygettext.py -d tinkerine -o tinkerine.pot !var!
+..\..\..\python\python.exe pygettext.py -d tinkerine -o tinkerine.pot !var!
```

Comment:
This has been change sine the script is made with the bundle python 2. My Python (3) installation could not handle the script.